src/fprime_gds/common/loaders/fw_type_json_loader.py

```diff
@@ -0,0 +1,61 @@
+"""
+fw_type_json_loader.py:
+
+Loads flight dictionary (JSON) and returns name based Python dictionaries of Fw types
+
+@author jawest
+"""
+
+from fprime_gds.common.loaders.json_loader import JsonLoader
+from fprime_gds.common.data_types.exceptions import GdsDictionaryParsingException
+
+
+class FwTypeJsonLoader(JsonLoader):
+    """Class to load python based Fw type dictionaries"""
+
+    TYPE_DEFINITIONS_FIELD = "typeDefinitions"
+    
+    # Dictionary mapping from JSON dictionary type qualified name to GDS key in config dictionary
+    FW_TYPE_NAME_MAPPING = {
+        "FwPacketDescriptorType": "msg_desc",
+        "FwChanIdType": "ch_id",
+        "FwEventIdType": "event_id",
+        "FwOpcodeType": "op_code",
+        "FwTlmPacketizeIdType": "pkt_id"
```

Comment:
Do you think it would help / be possible to rename to GDS keys to match the typenames in the JSON dictionary ?

Reply:
This ties back to the question about why so few elements in the JSON dict - can this go out of date if we decide we need to ingest `FwDpIdType` into the config or something?

Reply:
Good suggestion! I will make the GDS keys match the type names from the JSON dictionary. Added my response to your second question below.